### PR TITLE
Add foreman-tasks to PR scanner

### PR DESCRIPTION
**Pull request by iNecas**

### CODE REVIEW

**Comment by iNecas** [comment]
@domcleal after the merge, I would like to ask you also to add the webhook setting to foreman-tasks, as I don't seem to have permissions for that

File: puppet/modules/slave/templates/test_pull_requests_foreman_tasks.json.erb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            "test_prefix": "Test Results:",
+            "allow_multiple": true,
+            "run_without_trigger": true,
+            "branches": {
+                "*": {
+                    "jenkins_job_name": "test_plugin_pull_request",
+                    "downstream_job_name": "test_plugin_foreman_tasks",
+                    "build_token": "<%= jenkins_build_token -%>",
+                    "addtl_jenkins_params": {
+                        "foreman_tasks": {
```
Comment (domcleal) [inline puppet/modules/slave/templates/test_pull_requests_foreman_tasks.json.erb]: hyphen?

File: puppet/modules/slave/templates/test_pull_requests_foreman_tasks.json.erb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                }
+            },
+            "repo_to_orgs": {
+                "foreman-tasks": [
+                    "theforeman",
+                    "katello"
+                ]
+            },
+            "repo_to_pull_id_param": {
+                "foreman_tasks": {
```
Comment (domcleal) [inline puppet/modules/slave/templates/test_pull_requests_foreman_tasks.json.erb]: hyphen?

**Comment by iNecas** [comment]
@domcleal updated

**Comment by domcleal** [comment]
Thanks, it'll roll out in within an hour or so.  I'll add the hook in the meantime.

**Comment by iNecas** [comment]
Thanks @domcleal

**Pull request merged**